### ISSUE TITLE
Question about the two instances of install.packages

### ISSUE THREAD

**Issue by bneigher** [bakdata/aws-lambda-r-runtime#47]
Which does what exactly?  It's hard to follow what the difference is between this: 
https://github.com/bakdata/aws-lambda-r-runtime/blob/master/Dockerfile#L28
and this
https://github.com/bakdata/aws-lambda-r-runtime/blob/master/r/compile.sh#L42

What is the functional difference between the two?  Do they need to be in sync?  Is one a base and the second meant to be r layer derivative specific?

Love this repo so far - will be fun to chain sfn with this :D :D

**Comment by philipp94831** [comment]
Hi @bneigher,
basically there are two ways of creating the R runtime: compiling on EC2 and in docker. You can choose whatever suits you best although I would always recommend the docker approach. They should be equivalent. Maybe, this can be unified a bit more to be less confusing. Hope this answers your question

**Comment by bneigher** [comment]
@philipp94831 thanks for the fast response!

Yea I'm realizing that now, I'm sticking with the docker approach (having issues getting RJDBC installed properly.. but its a Dockerfile / context issue with missing .so files which is on me)
